### PR TITLE
[EVNT-525] Fix Compliance report URL

### PR DESCRIPTION
**Pull request by fellipeh**
The event of type 'Policy report failed to upload' does not have a
hyperlink, this commit fix it.

### CODE REVIEW

**Review by CodeHeeler — APPROVED** [review]
LGTM, thank you!

File: app/redhat_insights/default/data/ui/views/events.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@ index=redhatinsights AND bundle = "rhel"
...
 | eval time=strptime(timestamp, "%Y-%m-%dT%H:%M:%S.%3N")
-| eval url = case(event_type=="drift-baseline-detected", DRIFT_URL.'payload.baseline_id', event_type=="policy-triggered", POLICIES_URL.'payload.policy_id', event_type="compliance-below-threshold", COMPLIANCE_URL.'payload.host_id', !isnull('payload.rule_url'), 'payload.rule_url')
+| eval url = case(event_type=="drift-baseline-detected", DRIFT_URL.'payload.baseline_id', event_type=="report-upload-failed", POLICIES_URL.'payload.request_id', event_type=="policy-triggered", POLICIES_URL.'payload.policy_id', event_type="compliance-below-threshold", COMPLIANCE_URL.'payload.host_id', !isnull('payload.rule_url'), 'payload.rule_url')
```
Comment (vkrizan) [inline app/redhat_insights/default/data/ui/views/events.xml:19]: Where does it links to? Is it Payload Tracker?

**Review by vkrizan — CHANGES_REQUESTED** [review]

File: app/redhat_insights/default/data/ui/views/events.xml (hunk 1 above)
Comment (vkrizan) [reply app/redhat_insights/default/data/ui/views/events.xml:19]: It seems that Payload Tracker has the frontend internal only. Nevertheless, @fellipeh can you please check the generated URL?

**Comment by vkrizan** [comment]
@fellipeh `Policy report failed to upload` is that from Compliance or Policies?

**Comment by fellipeh** [comment]
@vkrizan Good catch!  

> @fellipeh `Policy report failed to upload` is that from Compliance or Policies?

According to our data, from `Compliance`
image

File: app/redhat_insights/default/data/ui/views/events.xml [hunk 2]
```diff
@@ -13,10 +13,11 @@ index=redhatinsights AND bundle = "rhel"
 | eval DRIFT_URL = "https://console.redhat.com/insights/drift/baselines/"
 | eval POLICIES_URL = "https://console.redhat.com/insights/policies/policy/"
 | eval COMPLIANCE_URL = "https://console.redhat.com/insights/compliance/systems/"
+| eval REPORT_URL = "https://console.redhat.com/insights/inventory/"
```
Comment (vkrizan) [inline app/redhat_insights/default/data/ui/views/events.xml:16]: `s/REPORT_URL/INVENTORY_URL/`

**Review by vkrizan — APPROVED** [review]

File: app/redhat_insights/default/data/ui/views/events.xml (hunk 2 above)
Comment (fellipeh) [reply app/redhat_insights/default/data/ui/views/events.xml]: Done!

**Pull request merged**